api/loader/getter.go

```diff
@@ -71,17 +71,21 @@ func getRemoteTarget(rs *remoteTargetSpec) error {
 		log.Fatalf("Error getting wd: %s", err)
 	}
 
+	httpGetter := &getter.HttpGetter{
+		Netrc: true,
+	}
```

Comment:
This is copied from how the default getters are defined https://github.com/hashicorp/go-getter/blob/7e54fdc668a3c829d0349517e4b4b74cb04060c0/get.go#L63-L65